### PR TITLE
update apollo federation version to 0.20.4

### PR DESCRIPTION
**Pull request by mhvenkat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "graphql": "^14.0.0"
+    "graphql": "^15.0.0"
```
Comment (brianjquinn) [inline package.json:33]: let's make this "^14.5.0 || ^15.0.0"

Comment (mhvenkat) [reply package.json:33]: Updated the graphql version as per your suggestion to "^14.5.0 || ^15.0.0"

Comment (brianjquinn) [reply package.json:33]: you actually updated the `graphql` devDependency. I was suggesting you change the peerDependency to the above string. The dev dependency should remain unchanged

**Comment by mhvenkat** [comment]
Hi Brian, I reverted back as per you suggestion on peer dependencies vs devDependencies

File: package.json [hunk 2]
```diff
@@ -30,14 +30,14 @@
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "graphql": "^14.0.0"
+    "graphql": "^14.0.0 || ^15.0.0"
```
Comment (brianjquinn) [inline package.json:33]: please revert this as well - it's clear from the failing unit test that if this package is used with graphql 15 (provided as a peer dependency by consumers) that something breaks. We would likely need to make some changes and fully understand what has changed between graphql 14 and 15 in order to confidently put graphql 15 as a peerDependency.

**Review by brianjquinn — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (mhvenkat) [reply package.json]: I have reverted back to graphql 14 for both the dev and peer dependency

**Review by brianjquinn — APPROVED** [review]
looks good - this will be cut as a patch release

**Comment by mhvenkat** [comment]
Thanks for merging. Now I see a new patch release https://github.com/ExpediaGroup/graphql-component/releases/tag/v2.1.1

**Pull request merged**